modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -183,12 +183,19 @@ const RichText = ({
 						onBlur={onBlur}
 						onChange={(content) => handleContentChange(content)}
 						onFocus={onFocus}
-						onSetData={({
-							data: {dataValue: value},
-							editor: {mode},
-						}) => {
-							if (mode === 'source') {
-								handleContentChange(value);
+						onSetData={(event) => {
+							const value = event.data.dataValue;
+							const editor = event.editor;
+
+							if (editor.mode === 'source') {
+								const filteredValue = value.replace(
```

Comment:
Hey @pedro-oliveira446 , we can rename a variable `filteredValue` since its objective is to avoid `XSS attacks` by removing the onerror attribute, we can choose a name that reflects the purpose of the variable. I thought about `sanitizedValue`, do you have any suggestions?

Reply:
I agree with you, this name is better.